radix-engine-tests/tests/metering.rs

```diff
@@ -66,19 +66,18 @@ fn test_basic_transfer() {
     // (cd radix-engine && cargo test --test metering -- test_basic_transfer)
     assert_eq!(
         15000 /* CreateNode */
-        + 325000 /* DropLock */
+        + 510000 /* DropLock */
```

Comment:
Do you know why we have a large bump in locking/unlock operations?

Reply:
Yup, all invocations to native components must go through all the pre-execution substate reads (e.g. read the PackageInfo of this component)